cpp/src/arrow/util/bit-util-test.cc

```diff
@@ -395,6 +395,51 @@ TEST(BitmapAnd, Unaligned) {
   }
 }
 
+TEST(BitmapOr, Aligned) {
```

Comment:
Similar to the above, it seems `BitmapAnd`, `BitmapOr` and `BitmapXor` are using exactly the same test harness, except with different result values.